<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>z_scan_widget</class>
 <widget class="QWidget" name="z_scan_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="z_scan_controls">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>162</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dwell Time (ms)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="z_scan_dwell_time_image_scan_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.000200000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z maximum(μm)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_scan_start_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z minimum(μm)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="z_scan_number_of_points_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="z_scan_maximum_spin_box">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="z_scan_minimum_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>-50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_peaks_push_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Peaks</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="peak_distance_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20000000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peak Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="z_max_position_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z(μm):</string>
    </property>
   </widget>
   <widget class="QLabel" name="z_counts_max_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>kCounts/s:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="z_scan_plot_frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="z_scan_toolbar_frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
